fdb-relational-core/src/main/antlr/RelationalParser.g4

```diff
@@ -131,7 +131,7 @@ columnType
     : primitiveType | customType=uid;
 
 primitiveType
-    : BOOLEAN | INTEGER | BIGINT | FLOAT | DOUBLE | STRING | BYTES;
+    : BOOLEAN | INTEGER | BIGINT | FLOAT | DOUBLE | STRING | BYTES | UUID;
```

Comment:
So if we have a syntax construct to generate a UUID, why do we need `STRING` -> `UUID` promotion?

Reply:
If I understand your concern correctly, we do not currently support have support for construct to generate UUID. hence, for simple (not prepared) statements, the constant value supplied for UUID is always a `String` that is promoted to `UUID` downstream. 

Even with the support, postgres allows String arguments anywhere UUID is expected.